ibis/backends/tests/test_export.py

```diff
@@ -156,3 +150,37 @@ def test_no_pyarrow_message(awards_players, monkeypatch):
     monkeypatch.setitem(sys.modules, "pyarrow", None)
     with pytest.raises(ModuleNotFoundError, match="requires `pyarrow` but"):
         awards_players.to_pyarrow()
+
+
+@pytest.mark.parametrize("limit", limit_no_limit)
+def test_table_to_parquet(tmp_path, backend, limit, awards_players):
+    outparquet = tmp_path / "out.parquet"
+    awards_players.to_parquet(outparquet, limit=limit)
+
+    import pandas as pd
```

Comment:
I think you can safely import pandas at the top, since we use it across the whole test suite.